build_tools/regression_build_test.sh

```diff
@@ -286,12 +271,10 @@ common_in_mem_args="--db=/dev/shm/rocksdb \
     --sync=0 \
     --verify_checksum=1 \
     --delete_obsolete_files_period_micros=314572800 \
-    --max_grandparent_overlap_factor=10 \
     --use_plain_table=1 \
     --open_files=-1 \
     --mmap_read=1 \
     --mmap_write=0 \
-    --memtablerep=prefix_hash \
```

Comment:
Question: why are we removing these two options?

Reply:
option `max_grandparent_overlap_factor` is removed. similar PR: https://github.com/facebook/rocksdb/pull/2015

`prefix_hash` doesn't support concurrent writes any more, or we need to set `Options.allow_concurrent_memtable_write` to false.